frontend/app/mappers/benefit-application-service-mappers.server.ts

```diff
@@ -217,14 +310,67 @@ function toRelatedPerson({ dateOfBirth, firstName, lastName, personRelationshipC
       },
     ],
     PersonRelationshipCode: {
-      ReferenceDataName: personRelationshipCode,
+      ReferenceDataName: 'Spouse',
     },
     PersonSINIdentification: {
       IdentificationID: socialInsuranceNumber,
     },
+    ApplicantDetail: {
+      ConsentToSharePersonalInformationIndicator: confirm,
+      AttestParentOrGuardianIndicator: undefined,
+      PrivateDentalInsuranceIndicator: undefined,
+      InsurancePlan: undefined,
+    },
   };
 }
 
+interface ToRelatedPersonDependentArgs {
+  children: ChildState[];
+}
+
+function toRelatedPersonDependent({ children }: ToRelatedPersonDependentArgs): RelatedPerson[] {
+  const childData: RelatedPerson[] = [];
+
+  children.forEach((child) => {
+    if (child.information) {
+      childData.push({
+        PersonBirthDate: toDate(child.information.dateOfBirth),
+        PersonName: [
+          {
+            PersonGivenName: [child.information.firstName],
+            PersonSurName: child.information.lastName,
+          },
+        ],
+        PersonRelationshipCode: {
+          ReferenceDataName: 'Dependent',
+        },
+        PersonSINIdentification: {
+          IdentificationID: child.information.socialInsuranceNumber ?? '',
+        },
+        ApplicantDetail: {
+          ConsentToSharePersonalInformationIndicator: undefined,
```

Comment:
I think this prop should be removed since we don't have that information for a child.

Reply:
Reworked RelatedPerson type/function so having undefined types is no longer necessary (removed the fields)